apis/placement/v1alpha1/common.go

```diff
@@ -17,4 +17,11 @@ const (
 
 	// OverrideClusterNameVariable is the reserved variable in the override value that will be replaced by the actual cluster name.
 	OverrideClusterNameVariable = "${MEMBER-CLUSTER-NAME}"
+
+	// OverrideClusterLabelKeyVariablePrefix is the reserved variable prefix in the override expression.
+	// The string will be replaced by the actual label value associated with the key following the prefix.
+	// The key name ends with a "}" character (but not include it).
+	// The key name must be a valid label name and case-sensitive.
```

Comment:
```suggestion
	// The key name must be a valid member cluster label name and case-sensitive.
```
indicate it's member cluster label?

Reply:
I think the emphasis here is on its format, I added more verbiage.